sdk/core/loader/boot.cc

```diff
@@ -405,36 +405,41 @@ namespace
 	 */
 	void *find_export_target(const ImgHdr &image,
 	                         const auto   &sourceCompartment,
-	                         ptraddr_t     target,
-	                         size_t        size)
+	                         ImportEntry  &entry)
 	{
 		// Build an MMIO capability.
 		auto buildMMIO = [&]() {
-			Debug::log("Building mmio capability {} + {}", target, size);
+			Debug::log("Building mmio capability {} + {} (raw: {})",
+			           entry.address,
+			           entry.size(),
+			           entry.sizeAndPermissions);
+			Debug::log("Permissions: {}", entry.permissions());
```

Comment:
Why two `Debug::log`s back to back?

Reply:
Because the second one was crashing in an earlier version. Probably not needed now.